tests/run.sh

```diff
@@ -17,6 +17,8 @@
 
 DIR=$(readlink -f $(dirname $0))
 
+pip install boto3 # FIXME
+
```

Comment:
This is not great; however, I understand it is required to make it work.

Reply:
Sorry, I forgot to put link in the PR, but I created one here: https://github.com/mendersoftware/mender-test-containers/pull/569
I will remove the top commit before I merge (once the mender-test-container is published).